resolwe/flow/management/commands/list_docker_images.py

```diff
@@ -33,14 +33,21 @@ def handle(self, *args, **options):
         if options['format'] != 'plain' and options['format'] != 'yaml':
             raise CommandError("Unknown output format: %s" % options['format'])
 
-        # Gather a list of all custom Docker requirements the processes are using
-        all_docker_requirements = [
-            p.requirements['executor']['docker']
-            for p in Process.objects.filter(requirements__icontains='docker')
-        ]
-
-        # Filter the list to include only unique images (also, the 'images' field is optional)
-        unique_docker_images = set(i['image'] for i in all_docker_requirements if 'image' in i)
+        # Gather only unique latest custom Docker requirements that the processes are using
+        # The 'image' field is optional, so be careful about that as well
+        unique_docker_images = set(
```

Comment:
For the sake of nicer print-outs, I would order images by name: `...sorted(set(...`.